rules/expiring-todo-comments.js

```diff
@@ -173,11 +174,7 @@ function semverComparisonForOperator(operator) {
 
 const create = context => {
 	const options = {
-		terms: [
-			'todo',
-			'fixme',
-			'xxx'
-		],
+		terms: ['todo', 'fixme', 'xxx'],
```

Comment:
In the future, don't do unrelated changes.

Reply:
Oh. Might have been the linter but might bad for leaving it, sorry. Won't happen again.